src/app/home-page/widget/featured-content/featured-news.component.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpClient } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+import { AlertService } from '../../../shared/alert/state/alert.service';
+import { Dockstore } from '../../../shared/dockstore.model';
+
+@Component({
+  selector: 'featured-news',
+  template: ` <div [innerHTML]="myExternalHTML"></div> `,
+})
+export class FeaturedNewsComponent implements OnInit {
+  // TODO should parameterize FeaturedContentComponent or something
+  public myExternalHTML = '';
+
+  constructor(private http: HttpClient, private alertService: AlertService) {}
+  ngOnInit() {
+    this.alertService.start('Retrieve featured content');
```

Comment:
"news" instead of featured content

Also, I see it was/is this way for featured content, but I would make it "Retrieving" instead of "Retrieve" ( everywhere else is "...ing").